addons/website_slides_survey/models/slide_slide.py

```diff
@@ -64,14 +64,15 @@ def _compute_slide_type(self):
             if slide.slide_category == 'certification':
                 slide.slide_type = 'certification'
 
-    @api.model
-    def create(self, values):
-        rec = super(Slide, self).create(values)
-        if rec.survey_id:
-            rec.slide_category = 'certification'
-        if 'survey_id' in values:
-            rec._ensure_challenge_category()
-        return rec
+    @api.model_create_multi
+    def create(self, vals_list):
+        slides = super().create(vals_list)
+        for slide, vals in zip(slides, vals_list):
+            if slide.survey_id:
+                slide.slide_category = 'certification'
+            if 'survey_id' in vals:
```

Comment:
I don't exactly see how we could have "slide.survey_id" but not "'survey_id' in vals" since it's the create, but I guess you just copied the previous implementation ^^